samples/android/Preference/NamedUserPreference.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
```

Comment:
Our SDK doesn't expose a `NamedUserPreference`, so I created this one. If we ever do add one to the SDK, we can remove this class.